Source/Urho3D/Math/Vector2.h

```diff
@@ -146,6 +149,15 @@ class URHO3D_API IntVector2
     /// Return as string.
     ea::string ToString() const;
 
+    /// Return 2D vector.
+    Vector2 ToVector2() const;
```

Comment:
We have both `Vector2(const IntVector2& vector)` and `IntVector2::ToVector2()` which sucks -- two functions doing the same. Please remove one or another, same for Vector3.

Reply:
fixed with https://github.com/rbfx/rbfx/pull/506